### ISSUE TITLE
[bug] translateCheckedException after upgrading to v1.3.2 (works fine with v1.3.1)

### ISSUE THREAD

**Issue by damienbeaufils** [hcoles/pitest#443]
Hello,

I try to upgrade pitest to the v1.3.2 version, but I encounter an error when running `pitest` task:
```
VM : OpenJDK 64-Bit Server VM
Vendor : Oracle Corporation
Version : 25.151-b12
Uptime : 648
Input -> 
 1 : -Dfile.encoding=UTF-8
 2 : -Duser.country=FR
 3 : -Duser.language=fr
 4 : -Duser.variant
BootClassPathSupported : true

        at org.pitest.util.Unchecked.translateCheckedException(Unchecked.java:20)
        at org.pitest.coverage.execute.DefaultCoverageGenerator.calculateCoverage(DefaultCoverageGenerator.java:103)
        at org.pitest.coverage.execute.DefaultCoverageGenerator.calculateCoverage(DefaultCoverageGenerator.java:49)
        at org.pitest.mutationtest.tooling.MutationCoverage.runReport(MutationCoverage.java:115)
        at org.pitest.mutationtest.tooling.EntryPoint.execute(EntryPoint.java:116)
        at org.pitest.mutationtest.tooling.EntryPoint.execute(EntryPoint.java:49)
        at org.pitest.mutationtest.commandline.MutationCoverageReport.runReport(MutationCoverageReport.java:87)
        at org.pitest.mutationtest.commandline.MutationCoverageReport.main(MutationCoverageReport.java:45)
Caused by: org.pitest.util.PitError: Coverage generation minion exited abnormally!

Please copy and paste the information and the complete stacktrace below when reporting an issue
VM : OpenJDK 64-Bit Server VM
Vendor : Oracle Corporation
Version : 25.151-b12
Uptime : 648
Input -> 
 1 : -Dfile.encoding=UTF-8
 2 : -Duser.country=FR
 3 : -Duser.language=fr
 4 : -Duser.variant
BootClassPathSupported : true

        at org.pitest.coverage.execute.DefaultCoverageGenerator.gatherCoverageData(DefaultCoverageGenerator.java:142)
        at org.pitest.coverage.execute.DefaultCoverageGenerator.calculateCoverage(DefaultCoverageGenerator.java:87)
        ... 6 more
```
Full build log available here: https://travis-ci.org/damienbeaufils/clean-architecture-demo/jobs/337914818

When reverting to v1.3.1, [it works fine](https://travis-ci.org/damienbeaufils/clean-architecture-demo/jobs/337914152).

How to reproduce this:
* Checkout this `pitest-1.3.2` branch: https://github.com/damienbeaufils/clean-architecture-demo/tree/pitest-1.3.2
* Run `./gradlew pitest`

Thanks!

**Comment by hcoles** [comment]
Looks like the junit5 plugin needs a change as a result of #438.

**Comment by hcoles** [comment]
Just released 0.4 of the junit 5 plugin which should fix the issue.

**Comment by damienbeaufils** [comment]
Works fine with pitest-junit5-plugin v0.4! Thanks!
